winml/dll/module.cpp

```diff
@@ -107,6 +109,13 @@ STDAPI DllGetExperimentalActivationFactory(void* classId, void** factory) noexce
       return 0;
     }
 
+    std::wostringstream learning_model_session_experimental_class;
+    learning_model_session_experimental_class << XSTRINGIFY(WINML_ROOT_NS) << ".AI.MachineLearning.Experimental.LearningModelSessionExperimental";
+    if (requal(name, learning_model_session_experimental_class.str())) {
```

Comment:
How about `if (name == XSTRINGIFY(WINML_ROOT_NS) ".AI.MachineLearning.Experimental.LearningModelSessionExperimental"sv)`?

Reply:
(It's shorter, avoids the heap allocations and the string_view comparison exits early if the length doesn't match)